Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1465,7 +1482,7 @@ def identityFS(wf):
 from  Configuration.PyReleaseValidation.upgradeWorkflowComponents import *
 
 defaultDataSets={}
-defaultDataSets['2017']='CMSSW_8_1_0_pre1-80X_upgrade2017_design_v4_UPG17-v'
+defaultDataSets['2017']='CMSSW_8_1_0_pre2-80X_upgrade2017_realistic_v1_UPG17realGT-v'
```

Comment:
yes, it is requested automatically but it is stack there:

https://cmsweb.cern.ch/phedex/prod/Data::Subscriptions#state=create_since%3D1458743912%3Bfilter%3D%2F*%2FCMSSW_8_1_0_pre2-80X_upgrade2017_realistic_v1_UPG17realGT-v*%2FGEN-SIM%3Bnode%3DT2_CH_CERN